index.html

```diff
@@ -1,29 +1,193 @@
+            </div>
+        </div>
+        <div class="work-card">
+            <img src="/images/project2.svg" alt="Multi-Post Stories" />
+            <div class="content">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="stats">
+                    <h3 class="client">CANOPY</h3>
+                    <div class="dot"></div>
+                    <h3 class="myrole">Back End Dev</h3>
+                    <div class="dot"></div>
+                    <h3 class="year">2015</h3>
+                </div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                </p>
+                <div class="languages">
+                    <div class="lang">html</div>
+                    <div class="lang">css</div>
+                    <div class="lang">javaScript</div>
+                </div>
+                <button class="details-btn" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="work-card">
+            <img src="/images/project3.svg" alt="Tonic" />
+            <div class="content">
+                <h1 class="project-title">Tonic</h1>
+                <div class="stats">
+                    <h3 class="client">CANOPY</h3>
+                    <div class="dot"></div>
+                    <h3 class="myrole">Back End Dev</h3>
+                    <div class="dot"></div>
+                    <h3 class="year">2015</h3>
+                </div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                </p>
+                <div class="languages">
+                    <div class="lang">html</div>
+                    <div class="lang">css</div>
+                    <div class="lang">javaScript</div>
+                </div>
+                <button class="details-btn" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="work-card">
+            <img src="/images/project4.svg" alt="Multi-Post Stories" />
+            <div class="content">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="stats">
+                    <h3 class="client">CANOPY</h3>
+                    <div class="dot"></div>
+                    <h3 class="myrole">Back End Dev</h3>
+                    <div class="dot"></div>
+                    <h3 class="year">2015</h3>
+                </div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                </p>
+                <div class="languages">
+                    <div class="lang">html</div>
+                    <div class="lang">css</div>
+                    <div class="lang">javaScript</div>
+                </div>
```

Comment:
- [ ] Please let's make these project technologies a `ul` list and each of them a `li` instead of using a `div` tag. The reason is that it's more semantically correct. This equally helps screen readers and other technologies that depend on correct semantics to work.